dev-libs/litehtml/litehtml-0.9.recipe

```diff
@@ -0,0 +1,66 @@
+SUMMARY="Lightweight HTML/CSS rendering engine"
+DESCRIPTION="
+Litehtml is lightweight HTML/CSS rendering engine. The main goal of the \
+litehtml library is to give the developers the easy way to show the HTML pages \
+in theirs applications."
+HOMEPAGE="http://www.litehtml.com/"
+COPYRIGHT="2013-2025 Aliaksei Chapyzhenka"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/litehtml/litehtml/archive/refs/tags/v0.9.tar.gz"
+CHECKSUM_SHA256="ef957307da15b1258a70961942840bcf54225a8d75315dcbc156186eba35b1a7"
+PATCHES="litehtml-0.9.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
```

Comment:
Add libVersion(Compat) here (libVersion is 0.0.0)

Reply:
libVersion is not defined :)

Reply:
Still missing libVersion/libVersionCompat

Reply:
I see it in the PR, to double check ?

Reply:
LOL my bad :)